kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -874,9 +875,15 @@ define([
             $.fn.DataTable.ext.search.pop();
 
             $.fn.DataTable.ext.search.push((_settings, _data, dataIndex) => {
+                if (_settings.nTable.id !== tableId) {
```

Comment:
Make sure we only apply this to the staging area datatable. https://datatables.net/manual/plug-ins/search